packages/@react-aria/virtualizer/src/ScrollView.tsx

```diff
@@ -129,9 +129,13 @@ function ScrollView(props: ScrollViewProps, ref: RefObject<HTMLDivElement>) {
     if (!dom) {
       return;
     }
+    // TODO: move this check to somewhere common so it can be shared? Double check that this is the route we wanna go
+    let isTestEnv = process.env.NODE_ENV === 'test';
```

Comment:
Are we opening the door for people to use `patch-package` to turn off virtualizer?